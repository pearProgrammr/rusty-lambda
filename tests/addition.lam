1 + 2 + 3 + 4;
a := 9 + 8;
b := 7 + 6;
a + b;
