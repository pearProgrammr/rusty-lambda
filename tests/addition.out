Num(10)
Num(30)
